frontend/src/components/layout/app-sidebar.tsx

```
@@ -0,0 +1,80 @@
+  const { toggleSidebar } = useSidebar();
+
+  return (
+    <Sidebar collapsible="icon">
+      <SidebarHeader>
+        <div className="flex items-center gap-2">
+          {/* Collapse button on left, aligned with menu icons below */}
+          <button
+            onClick={toggleSidebar}
+            className="flex h-8 w-8 items-center justify-center rounded-md hover:bg-sidebar-accent"
+            aria-label="Toggle Sidebar"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)